_data/icons.yml

```
@@ -6,13 +6,13 @@ calendly:
 dribbble:
 facebook:
 flickr:
-github: sylhare
+github: BedrockStreaming/tech.bedrockstreaming.com
```

Should we use the Bedrock open source github repo ?

---

```suggestion
github: BedrockStreaming
```
We could also do that ? 
What do you prefer ?

---

I think that I used the `github: BedrockStreaming` syntax somewhere in #338

---

@Slashgear here: https://github.com/BedrockStreaming/tech.bedrockstreaming.com/pull/338/files#diff-ed7257df23ca1e919e2ffedceb3adbf4917ed630b667c8ecd0f0fc0d17aa177cR11

because it was `github: M6Web` so I changed it for `github: BedrockStreaming`

feel free to update it there too if you change it here 🤷